Module1/todo_app/test_app.py

```diff
@@ -0,0 +1,57 @@
+from dotenv import load_dotenv, find_dotenv
+import os
+from flask.testing import FlaskClient
+from todo_app import app
+import pytest
+import requests
+
+@pytest.fixture
+def client():
+    # Use our test integration config instead of the 'real' version
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+
+    # Create the new app.
+    test_app = app.create_app()
```

Comment:
If you do change your `app.py` as suggested to delay when the lists get loaded, it's still the case that your app calls Trello as soon as it _starts_ rather than when it gets its first request. That's a nice pattern, but it does mean we'll need to get our patching in place earlier!

So for example here, we might want to get our stubbing set up before we create the app:
```python
    # is patching requests.**request** the right choice here?
    monkeypatch.setattr(requests, 'request', stub)
    test_app = app.create_app()
```